cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,39 @@
-/// <reference types='cypress' />
+/// <reference types= 'cypress' />
+import { userData } from '../support/generate';
 
 describe('Book Store app', () => {
-  before(() => {
-    
+  beforeEach(() => {
+    cy.viewport(1980, 1020);
   });
 
-  it('', () => {
-    
+  it('Login with credentials', () => {
```

Comment:
Login with credentials => should provide the ability to login with valid credentials

Reply:
the same for other test names